### PR TITLE
DOC-413: Update Java code examples for the quickstart

### PR DESCRIPTION
**Pull request by m-hulbert**
> _NOTE TO REVIEWERS_ - please **do not review PRs in the `DRAFT` state**, as the PR may change substantially before it is ready to review. Thanks.

## Description

This PR updates and fixes the code examples for Java in the quickstart. See [JIRA](https://ably.atlassian.net/browse/DOC-413) for further information.

## Review

To review, visit the [quickstart](https://ably-docs-pr-1168.herokuapp.com/quick-start-guide/) page and view it in Java.

### CODE REVIEW

File: content/root/quick-start-guide.textile [hunk 1]
```diff
@@ -93,20 +93,32 @@ blang[php].
...
   For Android apps:
 
   ```[java]
-  compile 'io.ably:ably-android:1.2.0'
+  dependencies {
+    implementation 'io.ably:ably-android:1.2.0'
+  }
+
+  repositories {
+  	mavenCentral()
```
Comment (marklewin) [inline content/root/quick-start-guide.textile:120]: The indentation needs fixing in the `repositories` section.

File: content/root/quick-start-guide.textile [hunk 2]
```diff
@@ -343,8 +354,11 @@ The following example code publishes a message to the @quickstart@ channel with
 bc[jsall](code-editor:quick-start-guide/send-message). var channel = ably.channels.get('quickstart');
 channel.publish('greeting', 'hello!');
 
-bc[java]. Channel channel = ably.channels.get('quickstart');
-channel.publish('greeting', 'hello!');
+bc[java]. Channel channel = ably.channels.get("quickstart");
+channel.publish("greeting", "hello", new CompletionListener() {
+  @Override
+  public void onSuccess() {
+    System.out.println("Message sent");
```
Comment (marklewin) [inline content/root/quick-start-guide.textile:361]: This code sample is incomplete. I believe it should read:

```
channel.publish("greeting", "hello", new CompletionListener() {
  @Override
  public void onSuccess() {
    System.out.println("Message sent");
  }. // <--
}); // <--
```

**Review by marklewin — CHANGES_REQUESTED** [review]

File: content/root/quick-start-guide.textile (hunk 1 above)
Comment (m-hulbert) [reply content/root/quick-start-guide.textile]: This was odd, as both `dependencies` and `repositories` were displaying as a single tab in my IDE. I've replaced the `repositories` with double spaces and checked in web view and that fixes it.

File: content/root/quick-start-guide.textile (hunk 2 above)
Comment (m-hulbert) [reply content/root/quick-start-guide.textile:361]: Ah, this was my mistake - I missed the error handling section here as well as the closing brackets.

**Review by marklewin — APPROVED** [review]
LGTM

**Pull request merged**